src/pages/settings.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function Dashboard() {
```

Comment:
Rename this one too?
```suggestion
export default function Settings() {
```

Reply:
LOL..Copy pasta..3 in a row 🤦‍♂